docs/using-airbyte/getting-started/oss-quickstart.md

```diff
@@ -0,0 +1,191 @@
+
+**2: Extract the archive**
+
+This will create a directory named abctl which contains the executable along with other needed files. 
+
+```bash
+tar -xvzf {name-of-file-downloaded.linux-*.tar.gz}
+```
+
+**3: Make the extracted executable accessible**
+
+This will allow you to run `abctl` as a command
+
+```bash
+chmod +x abctl/abctl
+```
+
+**4: Add `abctl` to your PATH**
+
+This will allow you to run `abctl` from any directory in your terminal. 
+
+```bash
+sudo mv abctl /usr/local/bin
+```
+**5: Verify the installation**
+
+```bash
+abctl --version
+```
+
+If this command prints the installed version of the Airbyte Command Line Tool, it confirm that you are now ready to manage a local Airbyte instance using `abctl`.
+
+
+</TabItem>
+<TabItem value="abctl-windows" label="Windows" default>
+
+**1: Download the latest release of `abctl` [here](https://github.com/airbytehq/abctl/releases)**
+
+**2: Extract the archive**
+- Right click the zip file you've downloaded and select `Extract All...`, then choose a destination folder. 
+
+This creates a folder called abctl containing the abctl executable and other reqired files. 
+
+**3: Add the executable to your PATH**
+- In the "System Properties" window (you can find this by searching for "enviornment variables" in the Start menu), click the `Environment Variables` button
+- Under System variables, find the path and click to `Edit`
+- Click `New` and add the path to the folder you extracted the abctl files to in the previous step. 
+- Click `OK` to save these changes. 
+
+**4: Open a new Command Prompt or PowerShell window**
+
+This is important because changes to your PATH will only take effect in a newly opened window. 
+
+**5. Verify the installation**
+
+```bash
+abctl --version
+```
+
+If this command prints the installed version of the Airbyte Command Line Tool, it confirm that you are now ready to manage a local Airbyte instance using `abctl`.
+
+
+</TabItem>
+
+</Tabs>
+
+## 2: Run Airbyte
+
+Ensure that Docker Desktop is up and running. Then, with abctl installed, the following command gets Airbyte running:
+
+```
+abctl local install
+```
+
+Your browser may open automatically to the Airbyte Application. If not, access it by visiting [http://localhost:8000](http://localhost:8000).
+
+When prompted for a username and password, enter the following default values: 
+- username: `airbyte`
+- password: `password`
+
+To set your own username and password, use command line flags or variables. For example, to set the username and password to foo and bar respectively, you can run the following command:
+
+```bash
+abctl local install --username foo --password bar
+```
+
+Or, if setting these values in the .env file, you'd add the following: 
+
+```
+ABCTL_LOCAL_INSTALL_PASSWORD=foo
+ABCTL_LOCAL_INSTALL_USERNAME=bar
+```
+
+After supplying username and password you'll see the Airbyte workspace. Using  this interface, you can set up and manage all your connections and move data with ease! 
```

Comment:
Wording nit: "After supplying a username and password, you'll see the Airbyte workspace. Using this..."